sdk/storage/azure-storage-file-share/azure/storage/fileshare/_share_client.py

```diff
@@ -415,7 +424,14 @@ def delete_share(
         timeout = kwargs.pop('timeout', None)
         delete_include = None
         if delete_snapshots:
```

Comment:
what is/was the default for this in python sdk ? looks like `false` but could you confirm that?

Reply:
Yes the default is `False` for deleting snapshots.